hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -531,6 +538,10 @@ private TokenKycStatus tksFor(boolean flag) {
 		return flag ? TokenKycStatus.Granted : TokenKycStatus.Revoked;
 	}
 
+	private TokenPauseStatus tpsFor(boolean flag) {
```

Comment:
This is confusing with `transaction per second`. Should rename it to `pauseStatusOf`